cloud-shell/cloudbuild.yaml

```diff
@@ -38,3 +38,4 @@ steps:
             echo "No cloud-shell directory in tag $$TAG"
         fi
     done
+logsBucket: 'gs://sandbox-cloud-build-logs'
```

Comment:
Where is this used and how was this bucket created?

Reply:
Cloud Build stores its logs in a bucket. We need to be able to read from the bucket to stream the logs from our GitHub Actions worker. It looks like the permission system is much simpler if we use a custom bucket, rather than a Google-managed one

I just created a standard empty bucket in the UI